lib/Doctrine/ODM/PHPCR/Mapping/ClassMetadata.php

```diff
@@ -1124,6 +1151,47 @@ public function getParentClasses()
     }
 
     /**
+     * Return the classes that are required by children of this document
+     *
+     * @return string[]
+     */
+    public function getChildClasses()
+    {
+        return $this->childClasses;
+    }
+
+    /**
+     * Set the classes that are required by children of this document
```

Comment:
Set the class names or interfaces that children of this document must be instance of.